holidays/countries/benin.py

```
@@ -0,0 +1,199 @@
+    """
+
+    country = "BJ"
+    default_language = "fr_BJ"
+    # %s (estimated).
+    estimated_label = tr("%s (estimé)")
+    # Order 76-35 of June 30, 1976.
+    start_year = 1977
+    supported_categories = (PUBLIC, WORKDAY)
+    supported_languages = ("en_US", "fr_BJ")
+
+    def __init__(self, islamic_show_estimated: bool = True, *args, **kwargs):
```

Pylint (3.3.7)

[warning] Keyword argument before variable positional arguments list in the definition of __init__ function

(W1113)